src/hooks/call/useCallTemplates.ts

```diff
@@ -19,6 +19,7 @@ export function useActiveTemplates(
     group: groupId,
   });
 
+  // TODO: Why we are calling getTemplates twice here? This should probably be refactored a bit and use only one call.
```

Comment:
I added a PR on this that makes it  more explicit on what the code is doing. It was between this solution, or one where we fetch all templates (active and archived) templates from server and then filter out templates specified in this hook parameters.